src/app/dev/DevToys.Tools/Tools/Formatters/FormattersGroup.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace DevToys.Tools.Tools.Formatters;
+
+[Export(typeof(GuiToolGroup))]
+[Name(PredefinedCommonToolGroupNames.Formatters)]
+internal class FormattersGroup : GuiToolGroup
+{
+    [ImportingConstructor]
+    internal FormattersGroup()
+    {
+        IconFontName = "DevToys-Tools-Icons";
+        IconGlyph = '\u0103';
```

Comment:
I missed updating the icon for the formatters group. I'll fix this before checking in.